src/main/java/org/alliancegenome/curation_api/interfaces/crud/ConditionRelationCrudInterface.java

```diff
@@ -28,6 +28,11 @@ public interface ConditionRelationCrudInterface extends BaseIdCrudInterface<Cond
     @JsonView(View.FieldsAndLists.class)
     public ObjectResponse<ConditionRelation> create(ConditionRelation entity);
 
+    @PUT
+    @Path("/")
+    @JsonView(View.FieldsAndLists.class)
+    public ObjectResponse<ConditionRelation> update(ConditionRelation entity);
+
```

Comment:
@cmpich I think you might need to create a custom view for this... might be why the IT's are failing.